src/main/java/com/gachigang/ontherun/config/handler/ApplicationExceptionHandler.java

```diff
@@ -1,88 +1,85 @@
 package com.gachigang.ontherun.config.handler;
 
-import com.gachigang.ontherun.common.ExceptionMessage;
-import com.gachigang.ontherun.model.dto.ErrorDetails;
-import jakarta.servlet.http.HttpServletRequest;
+import com.gachigang.ontherun.common.exception.NotFoundException;
+import com.gachigang.ontherun.common.exception.UserAlreadyExistException;
+import com.gachigang.ontherun.model.dto.ErrorMessage;
+import io.jsonwebtoken.JwtException;
 import jakarta.validation.ConstraintViolationException;
+import lombok.NonNull;
 import lombok.RequiredArgsConstructor;
 import lombok.extern.slf4j.Slf4j;
+import org.springframework.core.env.Environment;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.AccessDeniedException;
+import org.springframework.security.core.AuthenticationException;
 import org.springframework.web.bind.annotation.ExceptionHandler;
-import org.springframework.web.bind.annotation.ResponseStatus;
 import org.springframework.web.bind.annotation.RestControllerAdvice;
 import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
 
-import java.time.LocalDateTime;
+import java.util.Optional;
 
 /**
  * Exception handler for application.
  */
+@Slf4j
 @RestControllerAdvice
 @RequiredArgsConstructor
-@Slf4j
 public class ApplicationExceptionHandler extends ResponseEntityExceptionHandler {
```

Comment:
I believe we can make it more generic, we can store this messages in Exception class and here just log and return it, so we will have only one method here so far

Reply:
I don't get it.

Reply:
Lets take it offline

Reply:
Okey